docs/development.md

```diff
@@ -19,11 +19,44 @@
 2. After running this command, Windows may ask you for permissions related to firewall, select `allow` in this case.
 3. Run the farmer (do this in another terminal): `subspace-farmer-x86_64-*-snapshot farm`
 
+## to Build From The Source for Public Testnet (primarily for developers)
+
+1. Download the specific source code, example for now: `wget https://github.com/subspace/subspace/archive/refs/tags/snapshot-2022-mar-09.tar.gz`.
```

Comment:
You can also clone the repository using `git` and checkout on the latest snapshot

Reply:
Also it would be better not to mention specific snapshot here:
```suggestion
1. Download the latest released source code (from [release](https://github.com/subspace/subspace/releases) page) or just clone repo using `git` and checkout on the latest release
```